[Bug]: Cannot build `json-c`

### Problem description

I was building the `json-c` package, but it got error.
```sh
git clone https://github.com/termux/termux-packages.git termux/termux-packages
cd termux/termux-packages
./build-package.sh -I json-c
```
and show me this error:
![image](https://github.com/termux/termux-packages/assets/47179236/b50f6d8a-b895-4ff5-a95e-2f86b197420a)

```sh
/data/data/com.termux/files/home/.termux-build/json-c/src/json_util.c:63:35: error: a function declaration without a prototype is deprecated in all versions of C [-Werror,-Wstrict-prototypes]
const char *json_util_get_last_err()
                                  ^
                                   void
```

### What steps will reproduce the bug?
```sh
git clone https://github.com/termux/termux-packages.git termux/termux-packages
cd termux/termux-packages
./build-package.sh -I json-c
```
### What is the expected behavior?

Building `json-c` package successfuly without getting errors.

### System information

termux-info:

```sh
Termux Variables:
TERMUX_APK_RELEASE=F_DROID
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_APP_PID=5946
TERMUX_IS_DEBUGGABLE_BUILD=0
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.0
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://packages-cf.termux.dev/apt/termux-main stable main
Updatable packages:
All packages up to date
termux-tools version:
1.38.3
Android version:
12
Kernel build information:
Linux localhost 4.19.157-perf-g604b910ced3e #1 SMP PREEMPT Wed Aug 17 14:20:02 WIB 2022 aarch64 Android
Device manufacturer:
Xiaomi
Device model:
22021211RG
LD Variables:
LD_LIBRARY_PATH=
LD_PRELOAD=/data/data/com.termux/files/usr/lib/libtermux-exec.so
```

--- a/packages/json-c/build.sh
+++ b/packages/json-c/build.sh
@@ -8,3 +8,6 @@ TERMUX_PKG_SRCURL=https://s3.amazonaws.com/json-c_releases/releases/json-c-${TER
 TERMUX_PKG_SHA256=8e45ac8f96ec7791eaf3bb7ee50e9c2100bbbc87b8d0f1d030c5ba8a0288d96b
 TERMUX_PKG_BREAKS="json-c-dev"
 TERMUX_PKG_REPLACES="json-c-dev"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+-DDISABLE_WERROR=ON
+"

